packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -422,19 +421,19 @@ describe('Profiler', () => {
       const renderer = ReactTestRenderer.create(
         <React.unstable_Profiler id="test" onRender={callback}>
           <AdvanceTime byAmount={10}>
-            <AdvanceTime byAmount={10} shouldComponentUpdate={false} />
+            <AdvanceTime byAmount={13} shouldComponentUpdate={false} />
           </AdvanceTime>
         </React.unstable_Profiler>,
       );
 
       expect(callback).toHaveBeenCalledTimes(1);
 
-      advanceTimeBy(30); // 25 -> 55
+      advanceTimeBy(30); // 28 -> 58
 
       renderer.update(
         <React.unstable_Profiler id="test" onRender={callback}>
-          <AdvanceTime byAmount={10}>
-            <AdvanceTime byAmount={10} shouldComponentUpdate={false} />
+          <AdvanceTime byAmount={4}>
+            <AdvanceTime byAmount={7} shouldComponentUpdate={false} />
```

Comment:
This change was an unnecessary one but made the test a little stronger since 10&10 could give a false positive.